CGAL_Core/include/CGAL/CORE/CoreDefs.h

```diff
@@ -55,7 +55,7 @@
 #else // CGAL_HEADER_ONLY
 
   #define CGAL_GLOBAL_STATE_VAR(TYPE, NAME, VALUE)  \
-    CGAL_CORE_EXPORT extern TYPE NAME;                   \
+    CGAL_EXPORT extern TYPE NAME;                   \
```

Comment:
@afabri Do you remember having committed that change? It is dated May 16 2018, and it reverts #456.

See https://github.com/CGAL/cgal/pull/456#issuecomment-396274500

Reply:
I don't . sorry.